### ISSUE TITLE
Unable to scroll down from top if all visible groups collapsed

### ISSUE THREAD

**Issue by Kanegasi** [WeakAuras/WeakAuras2#3350]
### Description

Related/duplicate of #2472 but made a new ticket to follow template.

When scrolling down with the mouse wheel, Weakauras stops with the third aura at the top. When expanding either of the two visible groups, Weakauras then allows me to scroll the entire list.

No difference with any other groups expanded/collapsed later down the list.

When those groups are collapsed again, the scroll lock does not occur again until scrollbar returns to the top.

Clicking the scrollbar arrows or dragging the scrollbar box are not affected.

Resizing the window "clears" this lock, allowing scrolling to continue regardless of groups, but the lock happens again upon reloading or logging.

### WeakAuras Version

3.7.3

### World of Warcraft Flavor

Retail (Default)

### Tested with only WeakAuras

- [X] Yes
- [ ] No

### Lua Error

None

### Reproduction Steps

1. Open Weakauras after logging in or after a reload
2. Scroll down, it stops
3. Expand any visible group, either at the top or where the scroll stopped
4. Scroll can continue
5. Scroll back up, collapse visible groups, verify lock
6. Resize window
7. Verify scrolling with or without expanded groups

### Last Good Version

Unknown

### Export String

[WeakAuras WTF.zip](https://github.com/WeakAuras/WeakAuras2/files/7491066/WeakAuras.WTF.zip)

**Comment by InfusOnWoW** [comment]
I can't reproduce any issue with those settings. Can you record a video that shows the problem?

**Comment by Kanegasi** [comment]
![GIF 14-Nov-21 14-54-14](https://user-images.githubusercontent.com/2686302/141697614-c52ddd26-8ed0-4a85-a5aa-414da9e64521.gif)

While recording this gif, I disabled other addons. I discovered that the "lock" goes away completely after resizing the window and reloading, something I didn't try twice when exploring the bug with addons disabled. After further investigation, the issue comes from my personal addon forcing the option window size and position:

`if IsAddOnLoaded("WeakAuras") then
	WeakAurasSaved.frame={
		["height"]=700,
		["width"]=1400,
		["xOffset"]=0,
		["yOffset"]=-100,
	}
end -- default WeakAuras window`

Interestingly, the scroll lock does *not* occur if I force 900 height and 900 width, a size I decided to go with.

This may not actually be a bug with WeakAuras itself, possibly an issue with Blizzard scrolling frames containing a specific amount of objects and height. My apologies for not being thorough enough with my testing.
